api/src/main/java/ca/bc/gov/educ/api/gradstudent/model/dc/GradStudentRecordPayload.java

```diff
@@ -12,6 +12,7 @@ public class GradStudentRecordPayload {
     private String program;
     private String programCompletionDate;
     private String schoolOfRecord;
```

Comment:
Can we get rid of schoolOfRecord?

Reply:
No, this is DTO EDX is using.   We need a separate ticket to handle the changes in the shared interface between 2 systems.